commons/src/main/java/com/odysseusinc/arachne/execution_engine_common/descriptor/dto/RuntimeEnvironmentDescriptorsDTO.java

```diff
@@ -15,11 +18,24 @@ public RuntimeEnvironmentDescriptorsDTO(List<RuntimeEnvironmentDescriptorDTO> de
         this.descriptors = descriptors;
     }
 
+    public RuntimeEnvironmentDescriptorsDTO(boolean docker, List<RuntimeEnvironmentDescriptorDTO> descriptors) {
+        this.docker = docker;
+        this.descriptors = descriptors;
+    }
+
     public List<RuntimeEnvironmentDescriptorDTO> getDescriptors() {
         return descriptors;
     }
 
     public void setDescriptors(List<RuntimeEnvironmentDescriptorDTO> descriptors) {
         this.descriptors = descriptors;
     }
+
+    public boolean isDocker() {
+        return docker;
+    }
+
+    public void setDocker(boolean docker) {
```

Comment:
Defect: This is not set anywhere

Reply:
This was added due this comment: "EE api needs to be enchanced to allow datanode to find out if docker is enabled." mentioned in: https://bitbucket.org/Odysseus/ods-execution-engine/pull-requests/13#comment-453894355. Please see note.

Reply:
This does not address the defect. The value is not set and thus will not be sent back to datanode